packages/frontend-shared/src/components/StandardModalHeader.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    class="bg-white rounded-t flex border-b-1px border-b-gray-100 min-h-56px px-24px top-0 z-1 sticky items-center justify-between"
+    class="bg-white rounded-t flex border-b-1px border-b-gray-100 min-h-56px pr-14px pl-24px top-0 z-1 sticky items-center justify-between"
```

Comment:
why did we keep the pr at 14px and also since there is no ring anymore why don't we get back to having px-24px and remove the ring?